tracing/README.md

```diff
@@ -0,0 +1,37 @@
+# Tracing package
+
+`package tracing` provides an API wrapper around the tracing library
+[jaeger](https://github.com/uber/jaeger-client-go) that allows you to
+instrument operations in your application.
+Jaeger tracer can be configured with an optional logger that logs errors/spans
+and a stats reporter for emitting metrics.
+Note that the tracer is initialized by default with usage of UberFx modules.
+If you decide to use it standalone, read on for an example on how to initialize
```

Comment:
This "it" could mean UberFx, a module, or Jaeger. You could rephrase for clarity. Telling people to keep reading UberFx docs if they just wanna do tracing seems odd, but maybe "the tracer" means package tracing? Need to clarify.